### DISCUSSION TITLE
Is it possible to install obtanium using obtanium?

### DISCUSSION THREAD

**Discussion by mainrs** [ImranR98/Obtainium#1421] [Q&A] [answered]
I tried it out and it makes sense that it does not work. Maybe I have to do a special step to make this work?

**Comment by ImranR98** [comment]
It works just like with any other app. What error are you seeing? Assuming you're talking about updating the app and not first-time install of course.

**Reply by mainrs** [reply]
So, I clicked on `Add app` and used the search function to find it. After adding obtanium, it tells me that it is not installed yet on my system.

I installed obtanium through F-droid. Maybe that's the issue.

The error message I see is a dialog box with the title "Some Errors Occurred" and the description text "Conflict [Obtanium]" and a single action button called "Okay"

**Reply by ImranR98** [reply]
Yep, the fdroid and regular variants have different app ids.

**Reply by mainrs** [reply, marked as answer]
So I installed the GitHub version and it worked. The app was even listed from the start inside of Obtanium. Thank you!

**Reply by DwainZwerg** [reply]
Another option would have been to enter the F-Droid version (https://f-droid.org/en/packages/dev.imranr.obtainium.fdroid/) in Obtainium instead, so that an attempt is made to update with the F-Droid version instead, which would then also work.
